measurement/plugins/speedtestdotnet/tests/test_measurements.py

```diff
@@ -0,0 +1,164 @@
+import subprocess
+from unittest import TestCase, mock
+
+import speedtest
+
+from measurement.plugins.speedtestdotnet.measurements import SpeedtestdotnetMeasurement, SPEEDTEST_ERRORS
+from measurement.plugins.speedtestdotnet.results import (
+    SpeedtestdotnetMeasurementResult,
+)
+from measurement.results import Error
+from measurement.units import RatioUnit, TimeUnit, StorageUnit, NetworkUnit
+
+
+class SpeedtestdotnetTestCase(TestCase):
+    maxDiff = None
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.id = "1"
+        self.stdnm = SpeedtestdotnetMeasurement("1")
+        self.sample_results_dict_valid = {
+            'download': 93116804.64881887,
+            'upload': 19256654.06593738,
+            'ping': 17.054,
+            'server': {
+                'url': 'http://speedtest.syd.honestybox.com.au:8080/speedtest/upload.php',
```

Comment:
We probably don't want test URLs in the repo like this. In this case a fake URL should be used. It will also let us know if the test is trying to actually call out (which it should not)

Reply:
This is a fake url, would it be sensible to make it more obvious?

Reply:
Yeah I think so. Otherwise people might actually start to attempt to reach it :) 
Something like `http://fake.site:8080/speedtest/upload.php` is good as it is very obviously not real.